tests/type-destructors/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -51,10 +50,8 @@ type Obj_Prop_x = $PropertyType<Obj, \"x\">;
 (42: Obj_Prop_x);
 
 function foo(o: Obj): $PropertyType<Obj, \"x\"> {
-  if (false)
-    return o.x;
-  else
-    return 0;
+  if (false) return o.x;
+  else return 0;
```

Comment:
Doing that would require a modification of the AST. Since Prettier is just a code formatter, we won't be doing that. Instead, you should use `eslint --fix`, a JS codemod or a Babel plugin to do that.